index.html

```diff
@@ -23,165 +30,161 @@
+          </div>
+          <!-- Artwork 3 -->
+          <div class="Flashcard-3 ArtData worksection multiPost">
+            <h3 class="Txt ArtTxt textcolor workText1 CreteRoundfont">
+              Profesional Art Printing Data
+            </h3>
+            <p class="Txt textcolor workText2 Interfont">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="ARTtecnoBTN">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button type="button" class="PrjBTN Interfont">See Project</button>
+          </div>
+          <!-- Artwork 4 -->
+          <div class="Flashcard-4 ArtData worksection multiPost">
+            <h3 class="Txt ArtTxt textcolor workText1 CreteRoundfont">
+              Profesional Art Printing Data
+            </h3>
+            <p class="Txt textcolor workText2 Interfont">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="ARTtecnoBTN">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button type="button" class="PrjBTN Interfont">See Project</button>
+            <!-- Artwork 5 -->
+          </div>
+          <div class="Flashcard-5 ArtData worksection multiPost">
+            <h3 class="Txt ArtTxt textcolor workText1 CreteRoundfont">
+              Profesional Art Printing Data
+            </h3>
+            <p class="Txt textcolor workText2 Interfont">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="ARTtecnoBTN">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button type="button" class="PrjBTN Interfont">See Project</button>
+          </div>
+          <!-- Artwork 6 -->
+          <div class="Flashcard-6 ArtData worksection multiPost">
+            <h3 class="Txt ArtTxt textcolor workText1 CreteRoundfont">
+              Profesional Art Printing Data
+            </h3>
+            <p class="Txt textcolor workText2 Interfont">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="ARTtecnoBTN">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button type="button" class="PrjBTN Interfont">See Project</button>
+          </div>
+          <!-- Flash Cards end  -->
+        </div>
     <!-- About me -->
     <section class="MainAboutme">
       <div class="Aboutme">
```

Comment:
- [ ] Kindly make the header on the `About me` section bigger to match the Figma design.
- [ ] Kindly give the header and the paragraph some `margin-left` and `margin-top` to match the Figma design.

Your design
![image](https://user-images.githubusercontent.com/35535130/166123576-832643a6-6049-4a42-bb33-c9254e4f91f8.png)

Figma design
![image](https://user-images.githubusercontent.com/35535130/166123589-807810de-df54-40b6-84c5-21b07791f2d9.png)